src/routes/votings/[id]/+page.svelte

```diff
@@ -1,9 +1,495 @@
 <script lang="ts">
+	import {
+		Breadcrumb,
+		BreadcrumbItem,
+		Button,
+		Modal,
+		Tag,
+		TextInput,
+		ToggleSkeleton
+	} from 'carbon-components-svelte';
+	import { ArrowRight, Add, Search } from 'carbon-icons-svelte';
+	import VotingResultTag from '$components/VotingResultTag/VotingResultTag.svelte';
+	import DownloadData from '$components/DownloadData/DownloadData.svelte';
+	import BillCategoryTag from '$components/BillCategoryTag/BillCategoryTag.svelte';
+	import BillCard from '$components/BillCard/BillCard.svelte';
+	import { DefaultVoteOption, type CustomVoteOption, DefaultVotingResult } from '$models/voting.js';
+	import { onMount } from 'svelte';
 	export let data;
-</script>
 
-<a href="{data.voting.id}/votes">Votes page</a>
+	let open = false;
+	let selectedTab = '';
+	let selectedMenu = 'summary';
+	let isViewPercent = false;
+	let searchQuery = '';
+	let isMobile = false;
+
+	$: totalVote = data.results.reduce((acc, vote) => acc + vote.total, 0);
+
+	interface AnchorElement extends HTMLElement {
+		offsetTop: number;
+		offsetHeight: number;
+	}
+
+	function dateConvertor(date: Date) {
+		const convertedDate = Intl.DateTimeFormat('th', {
```

Comment:
can just return `Intl.DateTimeFormat`. No need to declare ` const convertedDate`